src/components/DataSession/OperationWizard.vue

```diff
@@ -91,6 +93,13 @@ const wizardTitle = computed(() => {
 
 function goForward() {
   if (page.value == 'select') {
+    // if there are no images for a filter required by the operation, do not proceed
+    for (const inputKey in selectedOperationInputs.value) {
```

Comment:
This will stop you from advancing with that operation, but really want you want is to grey out the "configure" button if that operation is selected so the user knows its not available. Or even better grey the button out and have some text in the description showing the unmet requirement. Perhaps this is a new story of functionality to add to the wizard of showing the requirements/preconditions that must be met for an operation to be selected.